policy/custom/definitions/policyset/LogAnalytics.bicep

```diff
@@ -771,6 +771,48 @@ resource policyset_name 'Microsoft.Authorization/policySetDefinitions@2020-03-01
           }
         }
       }
+      {
+        groupNames: [
+          'CUSTOM'
+        ]
+        policyDefinitionId: extensionResourceId(customPolicyDefinitionMgScope, 'Microsoft.Authorization/policyDefinitions', 'LA-Microsoft.AzureRecoveryVault-SiteRecovery')
+        policyDefinitionReferenceId: toLower(replace('Deploy Diagnostic Settings for Site Recovery Event to Log Analytics Workspace', ' ', '-'))
```

Comment:
Should be called `Deploy Diagnostic Settings for Site Recovery Events to Log Analytics Workspace`

Changed `Event` to `Events`